src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
```

Comment:
Let's create more descriptive variable name, for now it's not clear what should be stored in this variable.